python/CSXCAD/CSProperties.pyx

```diff
@@ -388,16 +389,30 @@ cdef class CSProperties:
         """
         return self.__CreatePrimitive(c_CSPrimitives.POLYHEDRON, **kw)
 
-    def AddPolyhedronReader(self, filename, **kw):
-        """ AddPolyhedronReader(filename, **kw)
+    def AddPolyhedronReader(self, file:Path, **kw)->CSPrimitives:
```

Comment:
I prefer to supply str? why not make this the default type? or at least both?

Reply:
Done.